code/__DEFINES/misc_defines.dm

```diff
@@ -167,8 +167,19 @@
 
 // Metal foam states
 // teehee no one will find these here
-#define MFOAM_ALUMINUM 	1
-#define MFOAM_IRON 		2
+#define METAL_FOAM_ALUMINUM 	1
+#define METAL_FOAM_IRON 		2
+
+// Defines for foam
+
+/// The chemicals in the foam (if any) will never react.
+#define FOAM_REACT_NEVER			(2<<0)
+/// Chemicals in the foam will only react when the foam dissipates.
+#define FOAM_REACT_ON_DISSIPATE		(2<<1)
+/// Chemicals in the foam will react while the foam is still processing.
+#define FOAM_REACT_DURING_SPREAD	(2<<2)
+/// Chemicals in the foam will react when the foam first reaches a tile.
+#define FOAM_REACT_BEFORE_SPREAD	(2<<3)
```

Comment:
are these meant to bitshift on 2?

Reply:
I forgor that we use 1 for our base :skull: